requirements/constraints.txt

```diff
@@ -12,6 +12,9 @@
 # This file contains all common constraints for edx-repos
 -c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt
 
+# latest version breaks the tests.
```

Comment:
Can you make sure this gets a ticket with details on the failure? (It can then be listed in the comment here.)